### PR TITLE
Add simple lint script for blog post

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: scripts/lint-blog.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+import path from "node:path";
+import fs from "node:fs";
+import createEsmUtils from "esm-utils";
+
+const { __dirname } = createEsmUtils(import.meta);
+
+const blogPostDir = path.join(__dirname, "../website/blog");
```
Comment (fisker) [inline scripts/lint-blog.js:9]: We can use `new URL("../website/blog/", import.meta.url)`.

File: website/blog/2023-11-13-curious-ternaries.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Please give the `--experimental-ternaries` option a try and let us know what you
 
 _For a quick tl;dr, [see the release post](/blog/2023/11/13/3.1.0)._
 
-<!-- truncate -->
+<!--truncate-->
```
Comment (fisker) [inline website/blog/2023-11-13-curious-ternaries.md:15]: Does the one with space work? If it does, maybe we should use `<!-- truncate -->` instead, we always add space to js comments, why not html comments.

File: scripts/lint-blog.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+import fs from "node:fs";
+
+const blogPostDir = new URL("../website/blog/", import.meta.url);
+
+const files = fs.readdirSync(blogPostDir);
```
Comment (fisker) [inline scripts/lint-blog.js:7]: Why not use async version? Just curious.

**Review by fisker — APPROVED** [review]

File: scripts/lint-blog.js (hunk 3 above)
Comment (sosukesuzuki) [reply scripts/lint-blog.js:7]: There is no reason..

**Pull request merged**